package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "docs",
+  "scripts": {
+    "lint": "yarn install && yarn lint-md && yarn lint-js && yarn lint-php && yarn lint-language",
+    "lint-fix": "yarn install && yarn lint-md --fix; yarn lint-js --fix; yarn lint-php --fix; echo 'Fixed all auto-fixable problems'",
+    "lint-md": "yarn install && markdownlint-cli2 en/**/*.md",
+    "lint-js": "yarn install && eslint en",
+    "lint-php": "composer install --prefer-dist --no-progress --ansi --no-interaction --no-scripts --no-plugins --optimize-autoloader && vendor/bin/mdphpcs en",
```

```suggestion
    "lint-php": "vendor/bin/mdphpcs en",
```

Seems super weird running `composer install` here.  Realistically there's only a few people who will ever run this on their locally so there's no need to be "helpful"

---

Done. I'll add some documentation about how to run linting since it's no longer just "run the lint command"